### PR TITLE
feat: run the generator – fewer oneOf types, 12/07/2020

### PR DESCRIPTION
**Pull request by grant**
Runs the generator.

This update should include:
- changes from the simpler union types in JSON schema
- uniform event type descriptions

### CODE REVIEW

File: src/main/java/com/google/events/cloud/audit/v1/DestinationAttributes.java [hunk 1]
```diff
@@ -47,8 +47,8 @@
     /**
      * The network port of the peer.
      */
-    public NumResponseItems getPort() { return port; }
-    public void setPort(NumResponseItems value) { this.port = value; }
+    public String getPort() { return port; }
```
Comment (jskeet) [inline src/main/java/com/google/events/cloud/audit/v1/DestinationAttributes.java:50]: I would really like us to be able to represent this as a `Long`. Yes, it's a string in the JSON itself, but a string that should always represent a 64-bit integer. As a user, I would *definitely* expect it to be a numeric type of some description. We don't need to fix this in this PR, but I really think we need a concerted look at the accuracy and usefulness of the results in all languages. (I think we need to look at numbers, bytes fields, timestamps, durations, Struct, and the various wrapper types.)

File: src/main/java/com/google/events/cloud/cloudbuild/v1/StepTimeout.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
  */
 public class StepTimeout {
```
Comment (jskeet) [inline src/main/java/com/google/events/cloud/cloudbuild/v1/StepTimeout.java:24]: If we get a new class every time a google.protobuf.Duration is used, that's going to be a terrible user experience.

**Review by jskeet — APPROVED** [review]
Happy for this to be merged as the current generator results, but I'm really concerned about the types here.

File: src/main/java/com/google/events/cloud/audit/v1/DestinationAttributes.java (hunk 1 above)
Comment (grant) [reply src/main/java/com/google/events/cloud/audit/v1/DestinationAttributes.java]: I fixed this issue in the last commit.

File: src/main/java/com/google/events/cloud/cloudbuild/v1/StepTimeout.java (hunk 2 above)
Comment (grant) [reply src/main/java/com/google/events/cloud/cloudbuild/v1/StepTimeout.java:24]: Started an email thread to talk about this concern.

**Pull request merged**